src/enums/media-type.ts

```diff
@@ -3,5 +3,6 @@ export const MediaType = {
   LIVE: 'Live',
   AUDIO: 'Audio',
   IMAGE: 'Image',
+  DOCUMENT: 'Doc',
```

Comment:
does this need to be document (is it compared to document somewhere ?) or it doesn't matter here ?

Reply:
good catch, that's enums that can be used in other repos (for now values hardcoded), so update required!